libs/npm/core/src/components/grid-table/Pager.tsx

```diff
@@ -82,9 +82,11 @@ export const Pager: React.FC<IPagerProps> = ({
 
   // only want to run on page load to set qty to stored cookie
   React.useEffect(() => {
-    if (qtyCookie !== null) setSearchParams({ ...searchParams, qty: qtyCookie });
-    if (searchParams.get('qty') !== pageSize.toString()) {
-      setPageSize(parseInt(searchParams.get('qty') ?? qtyCookie ?? '20'));
+    if (qtyCookie !== null && qtyCookie !== 'undefined' && qtyCookie !== searchParams.get('qty')) {
```

Comment:
Unfortunately that is what the cookie returns when nothing is stored in it (not sure if that makes sense)

Reply:
It returns a string `'undefined'`?  That doesn't sound correct.

Reply:
eg) When the user loads the site for the very first time it will check to see if there is a cookie stored for qty, it will return 'undefined' as it is the first time it has been run

Reply:
I'd put money down that the cookie does not return a string value 'undefined'.  I strongly suspect it something you're doing when you request the value.

Reply:
Oops sorry to clarify, I do have the cookie setup to take the search params and assign it to itself which is where the undefined originates. I could try to restructure

Reply:
Might have figured it out, looked again with fresh eyes lol. Give me a tad here!

Reply:
Think this is the culprit 
`  const qtyCookie = document.cookie
    .split('; ')
    .find((row) => row.startsWith('qty='))
    ?.split('=')[1];
`

Reply:
Which is non-existent before the cookie is set for the first time